src/services/codefixes/fixAddMissingMember.ts

```diff
@@ -209,4 +231,25 @@ namespace ts.codefix {
             changeTracker.insertNodeAtClassStart(classDeclarationSourceFile, classDeclaration, methodDeclaration);
         }
     }
+
+    function addEnumMemberDeclaration(changes: textChanges.ChangeTracker, checker: TypeChecker, token: Identifier, enumDeclaration: EnumDeclaration) {
+        /**
+         * create initializer only literal enum that has string initializer.
+         * value of initializer is a string literal that equal to name of enum member.
+         * numeric enum or empty enum will not create initializer.
+         */
+        const hasStringInitializer = some(enumDeclaration.members, member => {
+            const type = checker.getTypeAtLocation(member);
+            return !!(type && type.flags & (TypeFlags.StringLike | TypeFlags.Enum));

```

Comment:
i would just check for `stringLike` here, and not enum. it can be both string or number, and since we are not handling numbers, i would leave that part off.